src/com/company/Main.java

```diff
@@ -3,6 +3,90 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        sum1();     //Задача №1 - проверка результата:
+        sum2();     //Задача №2 - проверка результата:
+        sum3();     //Задача №3 - проверка результата:
+        sum4();     //Задача №4 - проверка результата:
+        sum5();     //Задача №5 - проверка результата:
+    }
+
+    //Задача №1:
+    private static void sum1() {
+        int clientOS = 1;                             //0 — iOS, 1 — Android
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        } else if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("К сожалению для вашей ОС существует только web версия сервиса");
+        }
+    }
+
+    //Задача №2:
+    private static void sum2() {
+        int clientOS = 1;                             //0 — iOS, 1 — Android
+        int clientDeviceYear = 2015;
+        if (clientOS == 1 && clientDeviceYear >= 2015) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        } else if (clientOS == 1 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        } else if (clientOS == 0 && clientDeviceYear >= 2015) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 0 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("К сожалению для вашей ОС существует только web версия сервиса");
+        }
+    }
+
+    //Задача №3:
+    private static void sum3() {
+        int year = 2021;
+        boolean leaYear = year % 4 == 0 && year % 100 > 0 || year % 400 == 0;
```

Comment:
```suggestion
        boolean leaYear = year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```
Тот же момент, но более явный